src/jit/codegenarmarch.cpp

```diff
@@ -567,6 +567,44 @@ void CodeGen::genSetRegToIcon(regNumber reg, ssize_t val, var_types type, insFla
     instGen_Set_Reg_To_Imm(emitActualTypeSize(type), reg, val, flags);
 }
 
+//---------------------------------------------------------------------
+// genSetGSSecurityCookie: Set the "GS" security cookie in the prolog.
+//
+// Arguments:
+//     initReg        - register to use as scratch register
+//     pInitRegZeroed - OUT parameter. *pInitRegZeroed set to 'false' if and only if
```

Comment:
set --> is set

Reply:
This method is not zero-ing initReg so it sounds strange to me that an out-parameter is named pInitRegZeroed. It also doesn't know or check that it holds zero on input. IMO, it would be clearer to name the argument pInitRegUsed and set it to true if initReg is used for setting the cookie.

Reply:
I agree that it's confusing. I might rename it `pInitRegModified`. Theoretically it wouldn't have to set it to false if it had actually zero'd the register, but that doesn't actually happen anywhere. It might be worth a comment here in the header to note that the caller is passing in the `initReg` that it uses in the prolog and therefore it needs to know whether it has been set to a non-zero value (or something to that effect).

Reply:
While I don't disagree about it being potentially confusing, note that `pInitRegZeroed` is the name used in many functions that do the same thing.

Reply:
Created issue https://github.com/dotnet/coreclr/issues/24985 to address this work